services/director-v2/requirements/_base.txt

```diff
@@ -58,6 +59,8 @@ anyio==3.2.1
     #   starlette
 asgiref==3.4.1
     # via uvicorn
+async-cache==1.1.1
```

Comment:
so the director-v2 already has aiocache installed (see above), do we really need yet another such library?

Reply:
hmm this got not reverted? forgotten?

Reply:
why do I still see async-cache here?

Reply:
forgot to update, removed it